scipy/stats/_stats_py.py

```diff
@@ -1013,6 +1013,14 @@ def _moment(a, moment, axis, *, mean=None):
         # Starting point for exponentiation by squares
         mean = a.mean(axis, keepdims=True) if mean is None else mean
         a_zero_mean = a - mean
+
+        eps = np.finfo(a_zero_mean.dtype).resolution * 10
```

Comment:
10? 100? 1000? How much precision loss is too much?

Reply:
I suppose we can start conservative like so and update it if it's too annoying for users.

Reply:
Mmm but 1 digit of loss is better than 2 no? That's what I interpreted as conservative

Reply:
Yes, but _all but one_ digit loss is worse than _all but two_ digits lost (15 digits lost vs 14 digits lost). A factor of 10 here corresponds with 15 digits lost.

Reply:
Yes ok we are talking about the same. Conservative for me meant more "be strict" as having a high number would not bother anyone and show any warnings.

Reply:
Ok. By "like so", I thought you meant "like it is now,  which very rarely emits the warning.
 In any case, if we were to be conservative by using, say, 10^6 here, the warning would be emitted whenever ~10 or more digits are lost, which is more frequent than as it is now: with 10^1 here, we emit only when 15 digits are lost. Where do you think the sweet spot is?

Reply:
Personally I never need that much precision (because I have so much uncertainty elsewhere that precision here would be artificial) so I am not sure. Maybe 10 digits is enough.

Reply:
Thanks @tupui I changed it to 1e6 so that we only warn if ~10 or more digits are lost.